### PR TITLE
Fix text break in Firefox

### PR DESCRIPTION
**Pull request by MartijnCuppens**
`word-break: break-word` doesn't work in Firefox, but I think `hyphens: auto;`  is an acceptable workaround. I guess `!important` won't be nessesary for `hyphens: auto;`, but not 100% sure.

### CODE REVIEW

**Comment by XhmikosR** [comment]
LGTM as a workaround. Maybe add a comment to revisit this.

**Comment by mdo** [comment]
Hmm, didn't know about this! [Could `overflow-wrap` work?](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap)

File: scss/utilities/_text.scss [hunk 1]
```diff
@@ -62,7 +62,10 @@
 
 .text-decoration-none { text-decoration: none !important; }
 
-.text-break { word-break: break-word !important; }
+.text-break {
+  word-break: break-word !important; // IE & < Edge 18
+  overflow-wrap: break-word !important;
```
Comment (XhmikosR) [inline scss/utilities/_text.scss]: Should we update the docs too?

Comment (MartijnCuppens) [reply scss/utilities/_text.scss]: Done!

**Comment by MartijnCuppens** [comment]
Forgot to answer on this ticket, but it looked like `overflow-wrap` was indeed the best solution. I've kept `word-break: break-word` for backwards compatibility.

**Review by XhmikosR — APPROVED** [review]

**Comment by adamwathan** [comment]
Too late but I think what you actually want is `word-wrap: break-word` for old IE support, `word-break` is a different property with subtly different behavior and `overflow-wrap` is the newer name for `word-wrap`.

image

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

**Pull request merged**